### ISSUE TITLE
Logo-Variants of the upcoming SvelteKit

### ISSUE THREAD

**Issue by vedam** [sveltejs/branding#6]
Hey all,
as discussed [here](https://github.com/sveltejs/svelte/pull/5638) and already posted on Discord channel, I wanted to add them here as an issue.

As soon as we have a decision here i will immediately submit the corresponding assets (.png and .svg) into a PR.

<img width="1083" alt="SvelteKit" src="https://user-images.githubusercontent.com/64245/98278502-fcb2d300-1f98-11eb-8225-b6a10702ce6c.png">

<img width="1093" alt="SvelteKit-outline" src="https://user-images.githubusercontent.com/64245/98278545-0b00ef00-1f99-11eb-856f-040bfc874a03.png">

**Comment by arxpoetica** [comment]
They both look good (I have a slight preference for the more "flat" look of the first one, which makes it more versatile).

The second one presents a problem w/ the outlines because it really won't scale well at small sizes.

**Comment by vedam** [comment]
@arxpoetica Yep, the small-size-problem depends really on how/where we'll use it.
- will it be in a section on the svelte-site?
- will we use it more as standalone, decoupled from the svelte-page?
Within the svelte-page, there's no need to couple/double with the chain-logo IMO.
Standalone, we probably should.

As of now, I'm not really sure where/how we mention/communicate it.
Is it a kind of "...powered by"? The magic/engine behind?

**Comment by vedam** [comment]
From discord (I would like to catch opinions in one place):

**@antony** mentioned
> what about if it was on its own with the Svelte logo
> what about right... right...
> if the chainlink-looking Svelte logo was somehow tweaked to look even more like chainlinks - and then that used to portray > that kit is a chain of links which turn svelte into a full stack serverless application...
> Svelte [ChainLinks] Kit

**Comment by vedam** [comment]
Not a bad idea, except that I would not actually change the chain links.  :grimacing: 
Probably you won't read SvelteKit anymore but Svelte "S" Kit.
And shouldn't Kit be in the front then? From the logic point of view?

**Comment by arxpoetica** [comment]
> depends really on how/where we'll use it

@vedam when preparing public-facing branding, important to remember that people will grab it and use it all over the place, even if brand guidelines are in place.

**Comment by arxpoetica** [comment]
> And shouldn't Kit be in the front then? From the logic point of view?

I don't understand this. I think where Rich landed is SvelteKit, so definitely after.

**Comment by dummdidumm** [comment]
I prefer the first one, looks slick.

**Comment by ehrencrona** [comment]
I also prefer the first, but I'd suggest trying a heavier font weight on the "kit" part. I think it would look more dynamic.
(and possibly making the color of "kit" a bit lighter, just to increase the contrast, also adding some dynamism)
